addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -673,10 +673,64 @@ export function getTraversedNodes(editable, range = getDeepRange(editable)) {
     do {
         node = iterator.nextNode();
     } while (node && node !== range.startContainer);
+    if (
+        node &&
+        node.nodeType === Node.ELEMENT_NODE &&
+        node.childNodes &&
```

Comment:
I wasn't sure but I just tested it: `node.childNodes` is a `NodeList` and `Boolean(node.childNodes)` is always `true`, even when there are no child nodes. Did you mean `node.childNodes.length`?